mesheryctl/internal/cli/root/system/check.go

```diff
@@ -303,7 +296,9 @@ func (hc *HealthChecker) runKubernetesAPIHealthCheck() error {
 			log.Warn("!! cannot query the Kubernetes API")
 			return nil
 		}
-		return ErrK8SQuery(err)
+		errMsg := fmt.Errorf("%s. Your %s context is configured to run Meshery on Kubernetes using the %s token",
```

Comment:
really not sure if this line should be added in errors from health checks. It shouldn't be added I feel because it's kinda irrelevant to healthchecks.

// @navendu-pottekkat

Reply:
Its a preflight check so I think it will help in debugging. In the [Issue](https://github.com/meshery/meshery/issues/4731) @leecalcote mentions the error message fromat to be `Your `$contextname` context is configured to run Meshery on Kubernetes using the `$tokenname` token. The Kubernetes cluster is not accessible. Please confirm that your cluster available and that the token is valid. See https://docs.meshery.io/installation/quick-start for additional instructions.`

Reply:
The error is for k8s availability. better to remove the part of it that concerns the token thing? i.e. `using the$tokenname token` and `and that the token is valid.`

Reply:
Yeah. I'll remove it